custom/enikshay/integrations/nikshay/field_mappings.py

```diff
@@ -150,9 +155,10 @@
 }
 
 health_establishment_type = {
-    'hospital': 'H',
-    'lab': 'L',
+    'Hospital': 'H',
+    'Lab': 'L',
     'pharmacy': 'P',
```

Comment:
is this supposed to not be capitalized. Its the only one

Reply:
there is actually no use for that value.. we can remove it later.